packages/web-runtime/src/store/app.ts

```diff
@@ -7,23 +7,24 @@ const state = {
 
 const actions = {
   showErrorMessage({ commit }, message) {
-    const getXRequestID = (error: AxiosError): string => {
-      if (!error) {
-        return
-      }
+    const getXRequestID = (error: AxiosError): string | null => {
       if (error.response?.headers?.['x-request-id']) {
         return error.response.headers['x-request-id']
       }
+      return null
     }
+
     message.status = message.status || 'danger'
     message.timeout = message.timeout || 0
+    message.errors = message.error ? [message.error] : message.errors || []
 
-    if (message.error) {
-      const xRequestID = getXRequestID(message.error)
-      if (xRequestID) {
-        message.errorLogContent = `X-Request-ID: ${xRequestID}`
-      }
-    }
+    const xRequestIds = message.errors
+      .map((error) => getXRequestID(error.reason || error))
```

Comment:
Reason comes from Promise.allSettled, this can be done in showErrorMessage fnc call as well. 

Opinions ?